app/controllers/concerns/multiple_destroy.rb

```diff
@@ -0,0 +1,28 @@
+module MultipleDestroy
+  extend ActiveSupport::Concern
+
+  def multiple_destroy
+    # cache these values
+    @count = params[:ids].size
+    @max_deleted_inline = ::Configuration.max_deleted_inline
+
+    if @count > 0
+      @job_logger = Log.new
+      job_params = {
+        author_email: current_user.email,
+        ids: params[:ids],
+        klass: item_class_name,
```

Comment:
here we could use something like: `params[:controller].singularize.capitalize`
and avoid these ìtem_class_name` methods